conf/ocsci/independent_rhcs_cluster.yaml

```diff
@@ -0,0 +1,12 @@
+DEPLOYMENT:
+  independent_mode: True
+  force_download_client: True
+
+ENV_DATA:
+  storage_cluster_name: 'ocs-external-storagecluster'
+
+INDEPENDENT_MODE:
+  # base64 encoded data of json output
+  # from exporter script
+  # https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/create-external-cluster-resources.py&sa=D&ust=1594358560646000&usg=AFQjCNHNqJas0w2FqtjZM5d0-9N38-MJCw
+  external_cluster_details: PLACE_HOLDER
```

Comment:
How are we going to provide this detail, considering the fact that it can be different when using different pools?

Reply:
@sidhant-agrawal 
If user is running locally then output from exporter script needs to be converted into base64 format and put it into this file, if its a jenkins run then user has to copy the same base64 output in a text box (this change is yet to be made). 
This needs to be followed till we implement RHCS deployment through ocs-ci.

Let me know if I am missing something.